src/com/mambu/apisdk/util/RequestExecutor.java

```diff
@@ -95,4 +98,24 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 	public String executeRequest(String urlString, Method method, ContentType contentTypeFormat)
 			throws MambuApiException;
 
+	/**
+	 * Executes a request for a given URL Executes a request with given url and specifying the contentType, with some
+	 * parameters, a request method and the API return format.
```

Comment:
The description doesn't seem to match with the actual parameters. The same in Javadoc section. In Javadoc part may be if you could also add the description for return in this execute